apps/ide/src/plugins/webida.preference/view-controller.js

```diff
@@ -255,6 +244,21 @@ define([
                 preferenceDlg.onHide();
                 module.isOpened = false;
             }
+
+            // add listener for Restore default and Apply button
+            preferenceDlg.own(
```

Comment:
@leechwin 
Please follow the above instruction @kyungmi has mentioned.
Thanks.